manual/photon.xml.template

```diff
@@ -30,9 +30,26 @@
             <Label>DNS Domain</Label>
             <Description>DNS Domain</Description>
         </Property>
-        <Property ovf:key="guestinfo.ntp" ovf:type="string" ovf:userConfigurable="true">
-            <Label>NTP</Label>
-            <Description>NTP Servers (space separated)</Description>
+        <Category>Proxy Settings (optional)</Category>
+        <Property ovf:key="guestinfo.http_proxy" ovf:type="string" ovf:userConfigurable="true">
+            <Label>HTTP Proxy</Label>
+            <Description>Enter HTTP Proxy Server followed by the port and without typing "http://" before. Example: "proxy.provider.de:3128"</Description>
+        </Property>
+        <Property ovf:key="guestinfo.https_proxy" ovf:type="string" ovf:userConfigurable="true">
+            <Label>HTTPS Proxy</Label>
+            <Description>Enter HTTPS Proxy Server followed by the port and without typing "https://" before. Example: "proxy.provider.de:3128"</Description>
+        </Property>
+        <Property ovf:key="guestinfo.proxy_username" ovf:type="string" ovf:userConfigurable="true">
+            <Label>Proxy Username (optional)</Label>
+            <Description>Username for the Proxy Server</Description>
+        </Property>
+        <Property ovf:key="guestinfo.proxy_password" ovf:password="true" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+            <Label>Proxy Password (optional)</Label>
+            <Description>Password for the Proxy User</Description>
+        </Property>
+        <Property ovf:key="guestinfo.no_proxy" ovf:type="string" ovf:userConfigurable="true">
+            <Label>No Proxy</Label>
+            <Description>No Proxy for e.g. your internal domain suffix. Comma seperated (localhost, 127.0.0.1, domain.local)</Description>
```

Comment:
`seperated` is miss-spelled, it should be `separated`

Reply:
Fixed it.
![image](https://user-images.githubusercontent.com/31652019/72572400-ae75fb80-38c2-11ea-8106-a35819e49989.png)